sql/log.cc

```diff
@@ -6649,7 +6656,7 @@ void THD::binlog_prepare_for_row_logging()
    Write annnotated row event (the query) if needed
 */
 
-bool THD::binlog_write_annotated_row(Log_event_writer *writer)
+bool THD::binlog_write_annotated_row(Log_event_writer *writer, bool skip_annotate)
```

Comment:
Ambiguous variable name (due to `--skip-annotate-row-events`, that I previously mentioned)